qt/Session.cc

```diff
@@ -751,6 +751,7 @@ void Session::refreshAllTorrents()
 void Session::initTorrents(torrent_ids_t const& ids)
 {
     refreshTorrents(ids, TorrentProperties::MainAll);
+    refreshDetailInfo(ids);
```

Comment:
Thank you for this PR! I love the idea of fixing this but would prefer another approach -- pulling down the detail info for every torrent basically means that `transmission-qt` is loading _everything_ on every torrent, including all the file paths, etc. I commented on this [in the Sept 2021 discussion](https://github.com/transmission/transmission/pull/1753#pullrequestreview-761141762):

> This can be quite expensive when dealing with seedboxes -- pulling down strings for every file in thousands and thousands of torrents.
>
> Are there other approaches we could do with this, e.g. making a round-trip to get the torrent's file list when 'Open' is clicked?

^ I still think that is a better approach since it will save a lot of startup bandwidth and runtime memory in `transmission-qt`.

Reply:
That was my first attempt but getting the info on demand seemed to not work for the first time you try it. I can dig a little deeper. My idea was that this at least only updates on startup, not every update.

Reply:
Hmm, probably `RpcResponseFuture` is not the way.. I'll read up a bit on the whole signal/slot pattern of Qt.